###O código desenvolvido permite compreender de forma prática a relação entre consumo e ocupação em ambientes híbridos, fornecendo dados quantitativos que fundamentam decisões de eficiência energética. Essa etapa é essencial para o desenvolvimento da solução completa do projeto, servindo como base para as próximas fases de otimização e automação inteligente do consumo de energia.

In [8]:
import pandas as pd

# Carregar os dados, especificando o separador como ponto e vírgula
dados = pd.read_csv("/content/escritorio_energia.csv", encoding='latin1', sep=';')

# Estatísticas básicas
media_consumo = dados["Consumo (kWh)"].mean()
media_ocupacao = dados["Ocupação (%)"].mean()
correlacao = dados["Consumo (kWh)"].corr(dados["Ocupação (%)"])

print("=== Análise de Consumo Energético ===")
print(f"Média de consumo: {media_consumo:.2f} kWh/dia")
print(f"Média de ocupação: {media_ocupacao:.1f}%")
print(f"Correlação consumo x ocupação: {correlacao:.2f}")

# Identificação de desperdício
limite_otimo = media_consumo * 0.85
dias_desperdicio = dados[dados["Consumo (kWh)"] > limite_otimo]
print(f"\nDias com consumo acima do ideal: {len(dias_desperdicio)} dias")

=== Análise de Consumo Energético ===
Média de consumo: 68.77 kWh/dia
Média de ocupação: 64.0%
Correlação consumo x ocupação: 0.99

Dias com consumo acima do ideal: 21 dias


A média de consumo diário foi de 68,77 kWh, enquanto a ocupação média foi de 64%, evidenciando que o escritório mantém um nível significativo de consumo mesmo em dias com menor presença de colaboradores.

O coeficiente de correlação de 0,99 demonstra uma forte relação linear positiva entre as duas variáveis: quanto maior a ocupação, maior o consumo de energia. Essa correlação próxima de 1 confirma que o uso da energia está fortemente atrelado à presença física no espaço, o que é coerente com o comportamento esperado de escritórios híbridos — iluminação, climatização e equipamentos são mais utilizados nos dias de maior comparecimento.